src/current/v23.2/admission-control.md

```diff
@@ -7,14 +7,36 @@ docs_area: develop
 
 CockroachDB supports an admission control system to maintain cluster performance and availability when some nodes experience high load. When admission control is enabled, CockroachDB sorts request and response operations into work queues by priority, giving preference to higher priority operations. Internal operations critical to node health, like [node liveness heartbeats]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#node-liveness-issues), are high priority. The admission control system also prioritizes transactions that hold [locks]({% link {{ page.version.version }}/crdb-internal.md %}#cluster_locks), to reduce [contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#transaction-contention) and release locks earlier.
 
+[XXX](): check in with #cc-serverless team on the conflicting Serverless statements in here.
```

Comment:
This is incorrect, since we look at AC behavior when investigating issues in serverless clusters.
@JeffSwenson

Reply:
thanks, I have removed the note that said "Admission control is not available for CockroachDB serverless clusters"